src/object/schema_manager.c

```diff
@@ -3032,6 +3117,144 @@ sm_is_system_class (MOP op)
   return sm_get_class_flag (op, SM_CLASSFLAG_SYSTEM);
 }
 
+/*
+ * sm_check_system_class_by_name () - Checks whether the class name is
+ *    the same as the system class name.
+ * return: int
+ * name(in): class simple name
+ */
+int
+sm_check_system_class_by_name (const char *name)
+{
+  // *INDENT-OFF*
+  static
+  struct system_class_list
+  {
+    const char *name;
+    int len;
+  }
+  system_classes[] =
```

Comment:
This is a local variable and it is always initialized whenever it's called. I think it looks better to move the local variable to a static global variable.